.dotnet/src/Custom/Chat/OpenAIChatModelFactory.cs

```diff
@@ -116,9 +116,14 @@ public static ChatInputTokenUsageDetails ChatInputTokenUsageDetails(int audioTok
 
     /// <summary> Initializes a new instance of <see cref="OpenAI.Chat.ChatOutputTokenUsageDetails"/>. </summary>
     /// <returns> A new <see cref="OpenAI.Chat.ChatOutputTokenusageDetails"/> instance for mocking. </returns>
-    public static ChatOutputTokenUsageDetails ChatOutputTokenUsageDetails(int reasoningTokenCount = default, int audioTokenCount = default)
+    public static ChatOutputTokenUsageDetails ChatOutputTokenUsageDetails(int reasoningTokenCount = default, int audioTokenCount = default, int predictionAcceptedTokenCount = default, int predictionRejectedTokenCount = 0)
```

Comment:
Parameter names need to be updated: predictionAcceptedTokenCount --> acceptedPredictionTokenCount, and the other one.